ksql-engine/src/test/java/io/confluent/ksql/ddl/commands/CreateSourceCommandTest.java

```diff
@@ -230,6 +233,51 @@ public void shouldBuildSerdeOptions() {
     assertThat(cmd.getSerdeOptions(), is(SOME_SERDE_OPTIONS));
   }
 
+  @Test
```

Comment:
can we add a QTT for this? we should be able to pass it in the property overrides for a command

Reply:
Just added one for this.